src/menus

```diff
@@ -25,7 +25,7 @@ p({
   },
   "parent":"admin"
 })
-p({"class":"foam.nanos.boot.DAONSpecMenu","id":"admin.data","label":"Data Management","handler":{"class":"foam.nanos.menu.ViewMenu","view":{"class":"foam.nanos.boot.DAOConfigSummaryView"}},"parent":"admin"})
+p({"class":"foam.nanos.boot.DAONSpecMenu","id":"admin.data","order":1,"label":"Data Management","handler":{"class":"foam.nanos.menu.ViewMenu","view":{"class":"foam.nanos.boot.DAOConfigSummaryView"}},"parent":"admin"})
```

Comment:
still need to review - menu orders

Reply:
done - coincides with nanopay pr. If you have admin.data menu this makes sense to be the first landing spot.